### PR TITLE
Ajd chapter13

### PR DESCRIPTION
**Pull request by alexdeas**
ajd chapter 13

### CODE REVIEW

File: answers.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+1) D I supposed, but does that mean code created before generics was never any of these things?
```
Comment (utopia-2) [inline answers.md:1]: :+1: You tell 'em.

File: src/Chapter13/Stackable.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package Chapter13;
+// Stack interface with a less rubbish name
```
Comment (utopia-2) [inline src/Chapter13/Stackable.java:2]: :+1:

File: answers.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+```java
+Counter someCount = new Counter( new String("Better than filling in blanks I guess") );
+```
+
+11) Type parameters are thrown away by the compiler so are long gone at run time.
+
+12) See the new stack classes.
+
+13) ``<>`` is the diamond operator, which lets you infer type parameters when instantiating new objects. Not nearly
+as cool as the diamond operator in Perl obviously.
```
Comment (codavex) [inline answers.md:55]: Obviously.

File: src/Chapter13/Stack.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  }
+  
+  public String toString () {
+    // Useful for dumping
+    String s = new String( );
+    for ( T object : this.stack ) s += object.toString();
+    return s;
+  }
+  
+}
```
Comment (codavex) [inline src/Chapter13/Stack.java:46]: :+1: looks good

File: answers.md (hunk 3 above)
Comment (Spinodal) [inline answers.md:55]: Nah its all about the diamond operator in SQL!

**Pull request closed without merging**